### PR TITLE
Update operand version to v0.6.0

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Also Add  missing nodeSelector for worker daemonSet 
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: manifests/0600_operator.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ spec:
             - name: OPERATOR_NAME
               value: "node-feature-discovery-operator"
             - name: NODE_FEATURE_DISCOVERY_IMAGE
-              value: "quay.io/kubernetes_incubator/node-feature-discovery:v0.5.0"
+              value: "quay.io/kubernetes_incubator/node-feature-discovery:v0.6.0"
```
Comment (marquiz) [inline manifests/0600_operator.yaml:39]: I think we could/should use `k8s.gcr.io/nfd/node-feature-discovery:v0.6.0`

**Review by marquiz — COMMENTED** [review]
Thanks! Could you squash these into one commit? Please also capitalize the first word of the message subject line

Good read 😄 
https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines

**Comment by ArangoGutierrez** [comment]
ups, Thanks @marquiz , will do

**Review by marquiz — COMMENTED** [review]
This probably sounds like useless nitting but seems like the commit message just hangs in the air unfinished 😄  "Use xxx instead..." of what(?) Prolly "use k8s.gcr.io instead of quay.io"? Also, you could comment on the nodeSelector why it is added in this commit.

**Comment by marquiz** [comment]
Thanks for your patience @ArangoGutierrez, looks good to me 😄 

/lgtm

**Pull request merged**